controllers/PollController.js

```diff
@@ -0,0 +1,81 @@
+var flash = require('../utilities/flash.js').flash;
+var Model = require('../models/models.js');
+
+module.exports = {
+ 
+ deletePoll: function(req, res){
+  var id= req.query.id;
+  Model.PollsModel.remove({_id:id}, function(err, result){
+   if (err) {
+    flash(req, 'danger', 'poll was not deleted', 'try again');
+    console.log ("I could not delete the poll!")}
```

Comment:
You need to send a response to the request, even though there are errors, otherwise the user will never get a response